mappings/net/minecraft/util/random/RandomGenerator.mapping

```diff
@@ -4,7 +4,10 @@ CLASS net/minecraft/unmapped/C_rlomrsco net/minecraft/util/random/RandomGenerato
 	COMMENT <p> This interface should be used instead of {@link java.util.Random} for random number generation.
 	FIELD f_dhenmqup GAUSSIAN_SPREAD_FACTOR D
 	METHOD m_afzjxjzq nextInt (I)I
-		ARG 1 bound
+		COMMENT @return a random value between {@code 0} (inclusive) and {@code bound} (exclusive)
+		COMMENT @throws IllegalArgumentException if {@code bound} isn't positive
```

Comment:
I like the renames, make sure to swap this javadoc to the `max` name!

Reply:
oops, fixed